dev/benchmarking/ddl/1_08_table_public_machine.sql

```diff
@@ -0,0 +1,60 @@
+/*
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+*/
+
+CREATE EXTENSION IF NOT EXISTS "citext";  -- type for case-insensitive text
+
+-- MACHINE
+CREATE TABLE IF NOT EXISTS public.machine
+(
+  machine_id SERIAL PRIMARY KEY
+  , machine_name citext NOT NULL UNIQUE
```

Comment:
Perhaps in the command-line I can ask for a prefix/suffix to make the machine name unique. I have made a note for this when I change the script to generate "machine.json" from shell to Python + 3rd-party libraries.

Reply:
I think the main concern here is that if different people accidentally ingest benchmark results with the same machine name, the error may be difficult to undo later. So finding a way to make the name unique, even something dumb (such as "take a physical network interface's MAC address", if easily doable), would be welcome to avoid such errors.

Reply:
Super. There is now a unique `mac_address` field in the `machine` table, and `machine_name` is not required to be unique.